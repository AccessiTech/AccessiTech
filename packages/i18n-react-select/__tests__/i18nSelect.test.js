import React from 'react';
import { render } from '@testing-library/react';
import {
  EN,
  namespace,
  I18nSelect,
} from '../src/i18nSelect';

describe('@accessitech/i18n-react-select', () => {
  it('Is defined', () => {

    expect(EN).not.toBe(undefined);
    expect(namespace).not.toBe(undefined);
    expect(I18nSelect).not.toBe(undefined);
  });

  it('Mounts without crashing', () => {
    const { container } = render(
      <I18nSelect
        currentLang={EN}
        languageKeys={[EN]}
        displayStrings={{ [EN]: 'En' }}
      />
    );
    expect(container).toMatchSnapshot();
  });
});
