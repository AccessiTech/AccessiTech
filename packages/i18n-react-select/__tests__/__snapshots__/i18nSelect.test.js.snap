// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@accessitech/i18n-react-select Mounts without crashing 1`] = `
<div>
  <div
    class="selector-container"
  >
    <select
      aria-label="Language Selection"
      class="language-select form-select"
    >
      <option
        class="language-select-option"
        selected=""
        value="en"
      >
        En
      </option>
    </select>
  </div>
</div>
`;
